### DISCUSSION TITLE
How to Contribute

### DISCUSSION THREAD

**Discussion by PISH-RUN** [PISH-RUN/kidneed-front#3] [Show and tell]
To start first watch the below video:

https://user-images.githubusercontent.com/97082639/152986606-bb6ba9ac-5c93-42f8-a32b-5f8eb347dd2b.mp4

**Comment by ali-shabani** [comment]
https://medium.com/dooboolab/quick-start-for-contributing-to-whatssub-with-forking-workflow-16c8c971adc5
